data/gettingstarted.html

```diff
@@ -1559,6 +1607,13 @@ <h3>&zwnj;Window shortcuts - numeric keypad&zwnj;</h3>
                   </td>
                   <td>&zwnj;Toggle maximize.&zwnj;</td>
                 </tr>
+                             <div class="1904 hidden">
+              <tr>
+               <td>
+                 <b>Find and change these shortcuts in the Settings App, in the Devices -> Keyboards section.</b>
```

Comment:
Nice idea - would be even better to include a button here to navigate direct i.e. run the command `gnome-control-center keyboard`

i.e. similar to https://github.com/UbuntuBudgie/budgie-welcome/blob/master/data/gettingstarted.html#L1016

The "cmd" command execute's this bit of code https://github.com/UbuntuBudgie/budgie-welcome/blob/master/budgie-welcome#L324 - so need to add a new command

    elif uri == 'keyboard':
        subprocess.Popen(['gnome-control-center', 'keyboard'])

Reply:
Added button, but can't get it to go to center. Thanks for idea.